Makefile.docs

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = sphinx-build-2
```

Comment:
Why is this necessary? Why doesn't just `sphinx-build` work?

Reply:
It runs sphinx-build in python-3 environment.

Reply:
and avocado does not have python-3 packages https://github.com/avocado-framework/avocado/issues/2024